core/chains/terra/monitor/prom.go

```diff
@@ -0,0 +1,21 @@
+package monitor
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var promTerraBalance = promauto.NewGaugeVec(
+	prometheus.GaugeOpts{Name: "terra_balance", Help: "Terra account balances"},
+	[]string{"account", "terraChainID", "denomination"},
+)
+
+func (b *balanceMonitor) updateProm(acc sdk.AccAddress, bal *sdk.DecCoin) {
```

Comment:
This is not covered by the test. How do we normally test prometheus?

Reply:
normally just in QA / locally running a node if possible. I think its ok to leave this to QA